docs/tutorials/advanced/classification_of_strategies.rst

```diff
@@ -24,7 +24,7 @@ This allows us to, for example, quickly identify all the stochastic
 strategies::
 
     >>> len([s for s in axl.strategies if s().classifier['stochastic']])
-    40
+    41
 
```

Comment:
No, this strategy has memory depth 2 as it needs to remember if a 'probe' was punished (so that it can be remorseful).